src/main/java/seedu/address/model/todolist/Priority.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.model.todolist;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the priority of a todoList event.
+ * Guarantees: immutable; is valid as declared in {@link #isValid(String)}
+ * Setting Rule: user can set 1 for High; 2 for Medium; 3 for Low
+ */
+
+public class Priority {
+
+    public static final String DEFAULT_PRIORITY = "1"; // default field name
+    public static final String MESSAGE_CONSTRAINTS =
+            "The valid user input for priority is p/[priority]."
+                    + "While [priority] is a integer from 1 to 3,"
+                    + "and 1 for high priority, 2 for medium priority, 3 for low priority";
+
+    /*
+     * There should be only one integer, and the integer should be 1, 2 or 3.
+     */
+    public static final String VALIDATION_REGEX = "\\d";
```

Comment:
the regex doesn't seem to check if its 1, 2, 3

Reply:
i think its sth like "[1-3]"

Reply:
I am just thinking that using regex to test whether it is a one-digit number or not.